packages/mui-docs/src/NProgressBar/NProgressBar.js

```diff
@@ -30,6 +30,57 @@ const muiNProgressPulse = keyframes`
   }
 `;
 
+const animation = css`
+  ${muiNProgressPulse} 2s ease-out 0s infinite
+`;
+
+const GlobalStyles = createGlobalStyle`
```

Comment:
styled-components `createGlobalStyle` util which we use in the `GlobalStyles` component, does not support callbacks that return string template (we use this for setting the default theme). However, for using animations/keyframes, there is a requirement to use the string template, so the fix in scenarios like this is to use the `createGlobalStyle` util directly. 

These changes will be reverted before the merge, I just wanted to validate that we can somehow make it work.